app/ts/simulation/services/SimulationModeEthereumClientService.ts

```diff
@@ -71,7 +71,7 @@ export function getInputFieldFromDataOrInput(request: { input?: Uint8Array} | {
 }
 
 export const simulateEstimateGas = async (ethereumClientService: EthereumClientService, simulationState: SimulationState | undefined, data: DappRequestTransaction): Promise<EstimateGasError | { gas: bigint }> => {
-	if (simulationState === undefined) return { gas: await ethereumClientService.estimateGas(data) }
+	//if (simulationState === undefined) return { gas: await ethereumClientService.estimateGas(data) } // commented out because of nethermind not estimating gas correctly https://github.com/NethermindEth/nethermind/issues/5946
```

Comment:
```suggestion
	// commented out because of nethermind not estimating gas correctly https://github.com/NethermindEth/nethermind/issues/5946
	//if (simulationState === undefined) return { gas: await ethereumClientService.estimateGas(data) }
```